src/css/imports/transitions.less

```diff
@@ -0,0 +1,55 @@
+.jwplayer.jw-skin-seven {
+    .jw-slider-container {
+        .jw-knob {
+            width: 0;
+        }
+        &:hover {
+            .jw-knob {
+                width: @thumb-size;
+            }
+        }
+    }
+    .jw-overlays {
+         opacity: 0;
+         transition: opacity 1s;
```

Comment:
Because you've declared that elements with the `.jw-overlays` class will transition opacity here, you do not need to add the `transition` style again below for more specific `.jw-overlays` rules.